src/facade/dragonfly_connection.h

```diff
@@ -14,11 +14,13 @@
 #include "base/io_buf.h"
 #include "util/connection.h"
 #include "util/http/http_handler.h"
+#include "util/tls/tls_engine.h"
```

Comment:
why are this changes in header needed now?

Reply:
The one below is needed, it's used in the interface and it was not included. It's a good practice to include headers that are part of your interface and avoid implicit includes